### DISCUSSION TITLE
V23Beta截图任务栏显示bug：控制中心未勾选时，任务栏图标仍然显示

### DISCUSSION THREAD

**Discussion by Aninana** [linuxdeepin/developer-center#4330] [Troubleshooting | 故障排查] [answered]
如图所示，系统是V23 Beta内测版
我在控制中心取消勾选了截图，然而任务栏仍显示图标，注销，重启，开关特效都试过了，无效：
![截图_VirtualBox Machine_20230510163340](https://github.com/linuxdeepin/developer-center/assets/130546027/738b01c0-a167-45d4-a546-57d62cda5c4c)
![截图_VirtualBox Machine_20230510163410](https://github.com/linuxdeepin/developer-center/assets/130546027/62b71b42-e07e-4e54-b37c-452de03fc4cd)

**Comment by Decodetalkers** [comment]
@tsic404 我记得这是特殊需求？那么那边干脆不要显示这个设置好了？

**Comment by BLumia** [comment, marked as answer]
- https://github.com/linuxdeepin/developer-center/issues/4239

是同一个问题，可以在上面这个 issue 里跟进。

**Reply by Aninana** [reply]
原来已经有同样的问题了啊。看来这个真的是bug了，等修复吧。谢谢了！
